bsconfig.json

```diff
@@ -16,6 +16,6 @@
   ],
   "suffix": ".bs.js",
   "namespace": true,
-  "bs-dependencies": ["reason-react"],
+  "bs-dependencies": ["reason-react", "bs-webapi"],
```

Comment:
Might need to take this dependency out since we're not rewriting the export image function yet